scripts/basemap/centers_uusimaa-2050.R

```diff
@@ -12,5 +12,13 @@ centers <- here::here("data", "Uusimaa-kaava-2050_Pisteet.gpkg") %>%
   dplyr::filter(sf::st_intersects(., sf::st_combine(zones), sparse = FALSE)) %>%
   sf::st_join(zones, join = st_within)
 
+centers <- centers %>%
+  dplyr::add_row(kohteenNimi = "Riihimäki", kuvaus = "Lisäys MAL 2023 -työhön", SIJ2019 = 23016, hs15 = FALSE, geom = st_sfc(st_point(c(1, 1)))) %>%
+  dplyr::add_row(kohteenNimi = "Lahti", kuvaus = "Lisäys MAL 2023 -työhön", SIJ2019 = 26003, hs15 = FALSE, geom = st_sfc(st_point(c(1, 1)))) %>%
+  dplyr::add_row(kohteenNimi = "Orimattila", kuvaus = "Lisäys MAL 2023 -työhön", SIJ2019 = 25002, hs15 = FALSE, geom = st_sfc(st_point(c(1, 1)))) %>%
+  dplyr::add_row(kohteenNimi = "Pasila", kuvaus = "Lisäys MAL 2023 -työhön", SIJ2019 = 255, hs15 = TRUE, geom = st_sfc(st_point(c(25496273, 6676350)))) %>%
+  dplyr::add_row(kohteenNimi = "Kalasatama", kuvaus = "Lisäys MAL 2023 -työhön", SIJ2019 = 306, hs15 = TRUE, geom = st_sfc(st_point(c(25498797, 6674913)))) %>%
+  dplyr::add_row(kohteenNimi = "Meilahti", kuvaus = "Lisäys MAL 2023 -työhön", SIJ2019 = 234, hs15 = TRUE, geom = st_sfc(st_point(c(25494848, 6675221))))
+
```

Comment:
What about Lohja and Porvoo?

Reply:
I am now reading all centers from Uusimaa 2050 -kaava and not just centers inside HS15 region. That way Lohja, Porvoo etc. are added. I will add a comment in the code!